src/data/security-quiz.ts

```diff
@@ -10,6 +28,24 @@ const securityQuiz = [
       "Free wifi hotspots are often targeted by hackers and can put you at risk of MitM attacks (Man in the Middle Attacks) where a hacker spies on your internet traffic, and may even modify it without you knowing.",
     Link: "https://www.freecodecamp.org/news/personal-digital-security-an-intro/",
   },
+  {
+    Question: "3.	 A form of encryption that allows users to perform binary operations on encrypted data without ever decrypting the data is known as?",
+    Answer: "Homorphic Encryption",
+    Distractor1: "Symmetric Encryption",
+    Distractor2: "Asymmetric Encryption",
+    Distractor3: "Advanced Encryption",
+    Explanation: "Homomorphic Encryption is a form of encryption that allows users to perform binary operations on encrypted data without ever decrypting the data.",
+    Link: "https://www.freecodecamp.org/news/introduction-to-homomorphic-encryption/",
+  },
+  {
+    Question: "Which of the following is a cryptographic network protocol for operating network services securely over an unsecured network?",
+    Answer: "SSH",
+    Distractor1: "SFTP",
+    Distractor2: "HTTPS",
+    Distractor3: "OSPF",
+    Explanation: "The SSH(Secure Shell Protocol) is a cryptographic network protocol for operating network services securely over an unsecured network.",
```

Comment:
This explanation comes from this [Wikimedia article](https://en.wikipedia.org/wiki/Secure_Shell). We have to be careful about using direct quotes from 3rd party resources without proper citation. 

 ```suggestion
    Explanation: "The SSH(Secure Shell Protocol) is a secure way for users to access a computer over an unsecured network.",
```